great_expectations/core/util.py

```diff
@@ -427,6 +431,13 @@ def convert_to_json_serializable(  # noqa: C901 - complexity 32
     if StructType is not None and isinstance(data, StructType):
         return dict(data.jsonValue())
 
+    if sqlalchemy and isinstance(data, TextClause):
```

Comment:
```suggestion
    if sqlalchemy and TextClause and isinstance(data, TextClause):
```

Reply:
Actually how about just
```suggestion
    if TextClause and isinstance(data, TextClause):
```

Reply:
Sounds good :)